int/api/myplugin/list.go

```diff
@@ -0,0 +1,39 @@
+package myplugin
+
+import (
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/massalabs/thyra/api/swagger/server/restapi/operations"
+	"github.com/massalabs/thyra/pkg/plugin"
+)
+
+func newList(manager *plugin.Manager) operations.PluginManagerListHandler {
+	return &list{manager: manager}
+}
+
+type list struct {
+	manager *plugin.Manager
+}
+
+func (l *list) Handle(param operations.PluginManagerListParams) middleware.Responder {
+	ids := l.manager.ID()
+
+	payload := make([]*operations.PluginManagerListOKBodyItems0, len(ids))
+
+	//nolint:varnamelen
+	for index, id := range ids {
+		//nolint:exhaustruct
+		payload[index] = &operations.PluginManagerListOKBodyItems0{
+			ID: ids[index],
+		}
+
+		info := l.manager.Plugin(id).Information()
+
+		if info != nil {
```

Comment:
This code seems similar to api/plugin/get.go
Isn't it ? maybe it sould be factorized.
And i think the Port of the plugin is a mandatory information that should be returned here